test/e2e/fixtures/connected-state/state.json

```diff
@@ -96,90 +101,83 @@
       "featureFlags": {
-        "showIncomingTransactions": true,
-        "transactionTime": false
+        "showIncomingTransactions": true
       },
-      "knownMethodData": {},
-      "participateInMetaMetrics": false,
-      "firstTimeFlowType": "create",
-      "currentLocale": "en",
+      "forgottenPassword": false,
+      "frequentRpcListDetail": [
+        {
+          "chainId": "0x539",
+          "nickname": "Localhost 8545",
+          "rpcPrefs": {},
+          "rpcUrl": "http://localhost:8545",
+          "ticker": "ETH"
+        }
+      ],
       "identities": {
         "0x5cfe73b6021e818b776b421b1c4db2474086a7e1": {
           "address": "0x5cfe73b6021e818b776b421b1c4db2474086a7e1",
+          "lastSelected": 1626907346643,
           "name": "Account 1"
         }
       },
+      "infuraBlocked": false,
+      "ipfsGateway": "dweb.link",
+      "knownMethodData": {},
       "lostIdentities": {},
-      "forgottenPassword": false,
       "preferences": {
+        "hideZeroBalanceTokens": false,
+        "showFiatInTestnets": false,
         "useNativeCurrencyAsPrimaryCurrency": true
       },
-      "completedOnboarding": true,
-      "metaMetricsId": null,
-      "ipfsGateway": "dweb.link",
-      "selectedAddress": "0x5cfe73b6021e818b776b421b1c4db2474086a7e1"
+      "selectedAddress": "0x5cfe73b6021e818b776b421b1c4db2474086a7e1",
+      "useBlockie": false,
+      "useLedgerLive": false,
+      "useNonceField": false,
+      "usePhishDetect": true,
+      "useStaticTokenList": false
     },
     "config": {},
     "firstTimeInfo": {
       "date": 1575697234195,
-      "version": "7.7.0"
+      "version": "10.4.0"
     },
-    "PermissionsController": {
-      "permissionsRequests": [],
-      "permissionsDescriptions": {},
-      "domains": {
+    "PermissionController": {
+      "subjects": {
         "http://127.0.0.1:8080": {
-          "permissions": [
-            {
-              "@context": ["https://github.com/MetaMask/rpc-cap"],
+          "origin": "http://127.0.0.1:8080",
+          "permissions": {
+            "eth_accounts": {
               "parentCapability": "eth_accounts",
               "id": "f55a1c15-ea48-4088-968e-63be474d42fa",
               "date": 1594348332268,
               "invoker": "http://127.0.0.1:8080",
               "caveats": [
                 {
-                  "type": "limitResponseLength",
-                  "value": 1,
-                  "name": "primaryAccountOnly"
-                },
-                {
-                  "type": "filterResponse",
-                  "value": ["0x5cfe73b6021e818b776b421b1c4db2474086a7e1"],
-                  "name": "exposedAccounts"
+                  "type": "restrictReturnedAccounts",
+                  "value": ["0x5cfe73b6021e818b776b421b1c4db2474086a7e1"]
                 }
               ]
             }
-          ]
+          }
         }
       }
     }
   },
   "meta": {
-    "version": 47
+    "version": 68
```

Comment:
As it turns out, migrations run for our fixture state, and migration 68 clobbered the new permissions state I defined in this fixture. Therefore, I had to bump this and update the fixture to match version 68 state well enough for tests to pass.

Reply:
Why update the fixtures at all? The e2e test fixtures are generally left as-is, because they help validate that the migrations are working properly.

Reply:
Yeah, I restored all fixtures.